bin/functions/install.sh

```
@@ -63,24 +63,21 @@ function install_jars_docs () {
 function install_local_spatial_plugin () {
-  # Install deps
-  mkdir -p $CMR_DIR/dev-system/resources/elasticsearch/es_libs/
-
-  printf "\naction install-es-deps\n"
-  (cd $CMR_DIR/es-spatial-plugin && \
-  lein install-es-deps && \
-  cp es-deps/cmr-es-spatial-plugin-deps-0.1.0-SNAPSHOT-standalone.jar $CMR_DIR/dev-system/resources/elasticsearch/es_libs/)
-
-  # Install plugin
+  # Clean and create plugin directory
   rm -rf $CMR_DIR/dev-system/resources/elasticsearch/plugins/cmr_spatial
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)